Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -796,6 +797,20 @@ - (NSDictionary *)getLatestReferringParams {
     return origSessionParams;
 }
 
+- (NSDictionary*) getLatestReferringParamsSynchronous {
+    if (!self.isInitialized &&
+        !self.preferenceHelper.shouldWaitForInit &&
+        ![self.requestQueue containsInstallOrOpen]) {
+		NSLog(@"[Branch] Warning: getLatestReferringParamsSynchronous called before session init.");
+		NSLog(@"[Branch] Make sure Branch `initSessionWithLaunchOptions:`, `handleDeepLink:`, or "
```

Comment:
should we return an empty dictionary here better than making the app wait forever in case of user error?